PHP/PHP Source.sublime-syntax

```diff
@@ -2578,17 +2578,17 @@ contexts:
         2: meta.regex.modifier.php
         3: punctuation.definition.string.end.php
       pop: 1
-    - match: \"
+    - match: /|"
       fail: branch-point-double-quoted
     - match: ''
       push: scope:source.regexp.php
       with_prototype:
-        - match: (?=/{{regex_modifier}}"|")
+        - match: (?=/|")
           pop: 1
-        - match: \s((#)[^"]*$\n?)
+        - match: (#)[^/"]*($\n?|(?=/))
```

Comment:
It appears the negative charset already includes a possible `\n` at eol, but excludes trailing `/` or `"`. Thus pattern can be simplified.

```suggestion
        - match: (#)[^/"]*
```